src/components/UserInfo/userinfo.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { UserLogin } from 'components/Login/login.js';
+
+
+export const UserInfo = (props) => {
+  return (
+    <article className ='user-info-article'>
```

Comment:
I'm wondering if this should be an aside intead of an article? We can discuss futher and decide then.